arch/posix/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 zephyr_cc_option_ifdef(CONFIG_LTO -flto)
 
 zephyr_compile_options(
+  -Wno-format-truncation
```

Comment:
our policy for REVERTME patches is that we need to file a bug on it and put the bug ID in the commit message

Reply:
In which file/line is this warning appearing and with what gcc version?

Reply:
done

Reply:
It just says "Tracked in issue" without an ID

Reply:
hmm, it was at the beginning of the line and was treated as comment, fixed now.